.github/workflows/dev-package-test.yml

```diff
@@ -10,6 +10,7 @@ on:
       - "yarn.lock"
       # This workflow file
       - ".github/workflows/dev-package-test.yml"
+      - "tests/config/install-prettier.js"
```

Comment:
This change is trigger for `dev-package-test` action. I'll remove this after CI works fine.

Reply:
We can keep it.